### PR TITLE
[SCRIPT] Add a way to run all benchmarks

### PR DESCRIPTION
**Pull request by whitneywhtsang**

### CODE REVIEW

File: scripts/test-triton.sh [hunk 1]
```diff
@@ -288,6 +293,24 @@ run_benchmark_attention() {
...
+      continue
+    fi
+    benchmark=$(basename -- "$file")
+    if [[ $benchmark = @("__init__.py"|"benchmark_driver.py"|"benchmark_testing.py") ]]; then
+      continue
+    fi
+    echo
+    echo "****** Running ${benchmark%.py} ******"
+    echo
+    python $TRITON_PROJ/benchmarks/triton_kernels_benchmark/$benchmark
```
Comment (pbchekin) [inline scripts/test-triton.sh:310]: Does this look simpler?
```suggestion
    python $file
```

File: scripts/test-triton.sh [hunk 2]
```diff
@@ -288,6 +293,24 @@ run_benchmark_attention() {
     python $TRITON_PROJ/benchmarks/triton_kernels_benchmark/flash_attention_fwd_benchmark.py
 }
 
+run_benchmarks() {
+  cd $TRITON_PROJ/benchmarks
+  python setup.py install
+  for file in $TRITON_PROJ/benchmarks/triton_kernels_benchmark/*.py; do
+    if ! [ -f "$file" ]; then
```
Comment (pbchekin) [inline scripts/test-triton.sh:300]: Why this check is needed?

**Review by pbchekin — APPROVED** [review]
Looks good, a couple of nits.

File: scripts/test-triton.sh (hunk 2 above)
Comment (whitneywhtsang) [reply scripts/test-triton.sh:300]: It is indeed not needed, removing.

File: scripts/test-triton.sh (hunk 1 above)
Comment (whitneywhtsang) [reply scripts/test-triton.sh:310]: good idea.

**Review by etiotto — APPROVED** [review]

**Pull request merged**